docs/book/driver-deployment/installation.md

```diff
@@ -149,7 +144,7 @@ Some of the parameters have been explained in the previous section for block vol
 
 The parameters grouped by `NetPermissions` are as follows:
 
-- `Ips` - defines the IP range or IP subnet to which these restrictions will be levied upon. The default value for `Ips` is "*" which means all the IPs.
+- `Ips` - defines the IP range or IP subnet to which these restrictions will be levied upon. The default value for `Ips` is "*" which means all the IPs. Make sure there are no clashes in the IP ranges across all the NetPermissions as this will lead to undefined behavior for the clashing IPs.
```

Comment:
How about also updating the known issues section?

Reply:
This issue is present in all 2.x releases. Do you want to update all 5 release pages with this issue?
Also, IMO this isn't exactly an issue. It is just a wrong configuration setting.

Reply:
We do not prevent either. So, technically a user can deploy our driver without realizing the IP clash issue and may find out that the pods are failing to come up due to mount issues.

Reply:
Done. Added it to known issues in all 2.x releases.